<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-N7K85DJ" name="Test_Case_Prod.HomePage_TC" tests="1" failures="0" timestamp="4 Aug 2021 13:03:44 GMT" time="123.239" errors="1">
  <testcase name="Home_Page" time="123.239" classname="Test_Case_Prod.HomePage_TC">
    <error type="java.lang.AssertionError" message="expected [Applied Financial Lines is bringing a powerful proposition to the European and Mideast Financial Lines markets?much-needed, first-rate capacity combined with a stellar team of underwriters.] but found [Applied Financial Lines is bringing a powerful proposition to the European and Mideast Financial Lines markets—much-needed, first-rate capacity combined with a stellar team of underwriters.]">
      <![CDATA[java.lang.AssertionError: expected [Applied Financial Lines is bringing a powerful proposition to the European and Mideast Financial Lines markets?much-needed, first-rate capacity combined with a stellar team of underwriters.] but found [Applied Financial Lines is bringing a powerful proposition to the European and Mideast Financial Lines markets—much-needed, first-rate capacity combined with a stellar team of underwriters.]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:176)
	at org.testng.Assert.assertEquals(Assert.java:186)
	at Pages.Home_page.Quote(Home_page.java:547)
	at Test_Case_Prod.HomePage_TC.Home_Page(HomePage_TC.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
    </error>
  </testcase> <!-- Home_Page -->
</testsuite> <!-- Test_Case_Prod.HomePage_TC -->
